src/views/virtualmachines/list/selectedVMs.ts

```diff
@@ -3,7 +3,9 @@ import { getName, getNamespace } from '@kubevirt-utils/resources/shared';
 import { isEmpty } from '@kubevirt-utils/utils/utils';
 import { signal } from '@preact/signals-core';
 
-export const selectedVMs = signal<{ name: string; namespace: string }[]>([]);
+export type VMIdentifier = { name: string; namespace: string };
```

Comment:
please avoid using acronyms like VM, I suggest something like VirtualMachineIdentifier.
This also the kind of type that can actually suit all k8s CRs, so consider making it more general :)